File: Modul 1. Variable, Tipe Data, dan Operator/PRAK110-2310817120003-Natalie_Grace_Katindagho/PRAK110-2310817120003-Natalie_Grace_Katindagho.py
from math import sqrt
alas = 5
tinggi = 12
luas = int(0.5*alas*tinggi) 
sisi_miring = int(sqrt((alas * alas) + (tinggi * tinggi)))
keliling = alas + tinggi + sisi_miring 
print("Diket :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm\n")
print(f"Jawab :")
print(f"Sisi A = {tinggi} cm")
print(f"Sisi B = {sisi_miring} cm")
print(f"Sisi C = {alas} cm")
print(f"Keliling = {keliling} cm")
print(f"Luas = {luas} cm ")

File: Modul 1. Variable, Tipe Data, dan Operator/PRAK110-2310817120003-Natalie_Grace_Katindagho/PRAK110-2310817120003-Natalie_Grace_Katindagho.c
#include <stdio.h>
#include <math.h>

void main()
{
    int alas = 5;
    int tinggi = 12;
    float luas = 0.5 * alas * tinggi;
    int sisi_miring = sqrt((alas * alas) + (tinggi * tinggi));
    int keliling = alas + tinggi + sisi_miring;
    printf("Diket :\n");
    printf("Alas = %d cm\n", alas);
    printf("Tinggi = %d cm\n\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", tinggi);
    printf("Sisi B = %d cm\n", sisi_miring);
    printf("Sisi C = %d cm\n", alas);
    printf("Keliling = %d cm\n", keliling);
    printf("Luas = %.f cm\n", luas);
}